dev/index.ts

```diff
@@ -59,25 +60,76 @@ global._commands = commands;
 global._Vim = Vim;
 
 let view
+let enableVim = true
```

Comment:
can we dispatch an effect to toggle the extension from the compartment rather than relying on globals?